daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/preprocessing/ValueTranslator.scala

```diff
@@ -193,7 +193,7 @@ private[lf] final class ValueTranslator(
                 // it's ok to do `{"a": 1, "a": 2}`, where the second occurrence would just win.
                 if (recordFlds.length != flds.length) {
                   typeError(
-                    s"Expecting ${recordFlds.length} field for record $tyCon, but got ${flds.length}"
+                    s"Expecting ${recordFlds.length} field(s) for record $tyCon, but got ${flds.length}"
```

Comment:
:+1:

Reply:
Unfortunately, it was too much noise in this PR stemming from this minor/insignificant change (a two stage fix of the Canton Ledger API tests, complex compatibility exclusions, etc.). Leaving it out. It should go in a different PR, if you like to address it @remyhaemmerle-da